packages/neuron-wallet/tests/setup.ts

```diff
@@ -0,0 +1,26 @@
+export const originalXMLHttpRequest = window.XMLHttpRequest
+
+export const mockedXMLHttpRequest = {
+  open: jest.fn(),
+  send: jest.fn(),
+  readyState: null,
+  responseText: null,
+}
+
+Object.defineProperty(mockedXMLHttpRequest, 'readyState', {
+  get: jest.fn(() => 4),
+  set: jest.fn()
+});
+Object.defineProperty(mockedXMLHttpRequest, 'responseText', {
+  get: jest.fn(() => JSON.stringify([])),
```

Comment:
What does `JSON.stringify([])` mean?

Reply:
It converts a JSON to a string. https://www.w3schools.com/js/js_json_stringify.asp

Reply:
I mean, why is a stringified `[]` returned

Reply:
No specific reasons. It is just a mocked return for all the XMLHttpRequests in the tests. Later we may want to change this value if it is necessary in some tests.